### PR TITLE
Updates salvaging system on large freighter, hopefully will add to other ships

### PR DESCRIPTION
**Pull request by Qustinnus**
:cl: Qustinnus
rscadd:  New loot system, currently only in place on Large Freighters. Probably unbalanced, blame hitler.
/:cl:

### CODE REVIEW

**Comment by Qustinnus** [comment]
I can't figure out mapmerging, I used FastDMM so I ASSUMED it would work.

File: code/game/objects/effects/spawners/lootdrop.dm [hunk 1]
```diff
@@ -160,11 +160,139 @@
...
+			/obj/item/weapon/gun/magic/staff/honk,
+			/obj/item/weapon/gun/projectile/shotgun/automatic/combat,
+			/obj/item/weapon/gun/projectile/revolver/golden,
+			/obj/item/weapon/gun/projectile/revolver,
+			/obj/structure/statue/bananium/clown,
+			/obj/structure/statue/gold/hop,
+			/obj/structure/statue/silver/janitor,
+			/obj/vehicle/janicart,
+			/obj/item/stack/sheet/plasteel/fifty,
+			/obj/item/stack/sheet/mineral/adamantine{amount = 10},
```
Comment (Weisyl) [inline code/game/objects/effects/spawners/lootdrop.dm]: Adamantine isn't even used for anything

Comment (Qustinnus) [inline code/game/objects/effects/spawners/lootdrop.dm]: I didn't know tbh. I was curious what it did.

**Comment by Qustinnus** [comment]
goodluck with that map im gonna go sleep :^)

**Comment by Qustinnus** [comment]
Tested ingame by the way, beside my map not being mapmerged properly and the computers somehow turning back into their real versions everything should be okie dokie. Loot spawns as expected and anyone who wants to can use effects/spawner/salvage to mark their salvage spawns. Use rare sparingly as it can contain items ranging from weapons to SM shards.

**Comment by CrAzYPiLoT-SS13** [comment]
Boy, you dun fucked it up. I think you should remake this PR and open a new one.

**Comment by Qustinnus** [comment]
fuk off fix my map loser

**Pull request merged**